packages/fe-container/src/components/top-bar-menu/TopBarMenu.less

```diff
@@ -32,19 +34,22 @@
     padding-right: @menu-entry-padding;
 
     &.active {
-      background: var(--microlc-primary-color-tint-89);
       margin-bottom: @menu-entry-active-border;
       box-shadow: 0 @menu-entry-active-border var(--microlc-primary-color);
 
       .topBarMenuEntry_text {
```

Comment:
you could change this to
.topBarMenuEntry_text, i {...} to avoid duplication with the next entry